committee-code-of-conduct/charter.md

```diff
@@ -16,6 +16,10 @@ involved, and that the process is never perfect by nature of its need to serve
 everyone in the community equally. That said, the committee is oriented toward
 the protection of our community and takes that duty seriously.
 
+## Accountability
+
+All participants in the project at all levels, from one-time GitHub commenter to Steering Committee member, are accountable to the code of conduct and any actions the Code of Conduct Committee decides to take. 
```

Comment:
```suggestion
All participants in the project at all levels, from one-time GitHub commenters to Steering Committee members, are accountable to the code of conduct and any actions the Code of Conduct Committee decides to take. 
```

Project-wide accountability to the Code of Conduct makes sense.

"any actions the Code of Conduct Committee decides to take" needs some scoping/clarification :)

Reply:
Suggest cap's, ie: "...are accountable to the Code of Conduct..."

Reply:
@liggitt re: "any actions the Code of Conduct Committee decides to take" -> is there anything in specific you're concerned about here? How would you like to see this scoped further?

Reply:
(to start, I'll note that my concerns are not a judgment of past actions, but are forward-looking, wanting to make sure the boundaries we end up with have checks and balances and are resilient)

I'm strongly in favor of accountability for the bodies in our community. Because of that, I'm concerned that this statement, combined with significant ambiguity in the CoCC charter and incident response process, makes the CoCC entirely _unaccountable_ over specific decisions/actions. It's not clear from the charter or the incident response doc what the boundaries of the CoCC are:

* The guiding inputs [are not bounded](https://github.com/kubernetes/community/blob/master/committee-code-of-conduct/charter.md#missionpurpose) (emphasis mine):

    > The Code of Conduct serves as the primary policy document and is supported _with additional references and tools as needed_.

* The scope of jurisdiction [is not bounded](https://github.com/kubernetes/community/blob/master/committee-code-of-conduct/incident-process.md#when-and-where-does-the-kubernetes-code-of-conduct-apply) (emphasis mine):
    > The Code of Conduct applies between all community members _when interacting about Kubernetes_. [...]
    >
    > What are the boundaries of the Kubernetes community?
    > _There are no hard boundaries of the community_, [...]
    > 
    > * Social media
    >     In some cases, where individual social media messages _are not related to Kubernetes_ but have been reported to the Code of Conduct Committee and are making project members feel unsafe or unwelcome, we might choose to act.

* The actions the CoCC can decide to take [are not bounded](https://github.com/kubernetes/community/blob/master/committee-code-of-conduct/incident-process.md#deciding-on-a-course-of-action), either in the breadth of actions available, or allowed actions for specific categories of offenses. 
* Finally, there is no described way to appeal CoCC decisions/actions.

With that degree of ambiguity and lack of accountability around the CoCC, adding a blanket statement that community members are accountable to "any actions the Code of Conduct Committee decides to take" seems unwise to me.
 
I agree this would be a good topic for in-person discussion at the next Steering/CoCC sync, to think through how we can clarify this for community members and ensure all bodies have accountability.

Reply:
@liggitt Excellent observations! 

1. I agree that the guiding inputs are not bounded: having not been around when the committee was chartered, I'm not sure what was meant by "additional references and tools as needed". Perhaps @jdumars @tashimi or @AevaOnline know more and can shed some light?
2. The scope of jurisdiction is, I think, _necessarily_ unbounded: if something untoward happened in a KubeCon hotel room after KubeCon hours, would you want the people involved to have access to some kind of process? If so, the borders of where the CoC applies need to necessarily be fuzzy. Any attempt to define them clearly ("Only things that happen  _at_ KubeCon!!") takes away the ability of someone to seek out recourse. 
3. The actions the CoCC can decide to take are, again, _necessarily_ unbounded in my opinion: I think any attempt to ascribe what actions the CoCC can take puts us on a road to an [enforcement ladder](https://www.contributor-covenant.org/version/2/1/code_of_conduct/), which is something the CoCC has consciously and consistently tried to avoid:  if we say the CoCC can (i.e.) ban someone, warn them, etc. then whether we intend to or not those become the prescribed actions, and we end up warning someone ("Don't be rude to people!!!") instead of coaching them ("Can you see why that comment might've been poorly received? Do you think an apology might work?"). 
4.  Something brought to my attention recently through conference talks is the idea of building in an appeals process. You're right, the CoCC doesn't have this currently and I think it would be a useful next step.

Reply:
Re: "additional references and tools as needed" for @liggitt @celestehorgan &all...

In this context, the Code of Conduct document can be considered "a reference" and "a tool". This statement, as I understand the history, is open ended because it is understood that the Code of Conduct _is not the only tool_ available to the CoCC or to the community. While it is the authoritative document establishing a common baseline, as any governance document, it may need to be updated over time, additional guides created, and additional knowledge brought to bear to address previously-unconsidered circumstances. One would hope such updates are infrequent, and it is clear from the initial chartering of the CoCC that its bootstrap team intended for the CoC (and associated documents) to evolve over time.

Here are some examples (non-exhaustive list) of other tools that I considered supportive _but not authoritative_ during my term: books and written guides on the topics of community moderation and restorative justice; training performed or provided by recognized educators; and legal counsel.

Here is an example of an additional tool that _is_ authoritative and demonstrates this evolution: the incident reporting process (k8s.dev/coc-process) incrementally improves the tooling available within this community by adding to the baseline CoC document, informing all community members as to aspects of the process which the CoCC follows, and outlining that process for future CoCC members.

A document outlining an appeals process might be another such 'reference and tool' that the CoCC could create, for example.

Reply:
+1 on @celestehorgan 's points 2 and 3:

2. The scope of jurisdiction is intentionally left open so as to protect community members regardless of where (or on what platform) the community is gathering. Imagine if the initial framing had said the CoC only applies on Hangouts and Slack -- how would we have responded if there were incidents on Zoom or Twitch, platforms that weren't popular five years ago?

3. The potential remedies to an incident are also intentionally open ended, with training provided to committee members that centers on restoration of trust and emotional safety within the community. 

I continue to advocate _against_ embedding punitive ladders in CoC documents, as doing so would bind the Committee's potential actions and is likely to necessitate punishments in situations where the committee members do not feel they are appropriate. In my experience, codifying a punitive ladder can inadvertently create a system that is prone to gamification, less forgiving of accidents, and less conducive to either personal growth or restoration of trust after an incident occurs.

Reply:
@liggitt would changing the wording around the "additional references and tooling as needed" to something more specific allow us to keep scope of jurisdiction and remedies open? i.e.

> The Code of Conduct serves as the primary policy document. As needed, the code of conduct committee utilizes legal counsel, training provided to it via the CNCF's budget for the Kubernetes project, and other resources on community management.

Reply:
I appreciate the goal of starting with coaching/encouraging in the right direction, etc, but that doesn't rule out clearly stating _upper_ boundaries on actions for types of incidents (not required _minimum_ actions). Let's talk about this aspect when we meet.